pinot-connectors/pinot-flink-connector/src/main/java/org/apache/pinot/connector/flink/sink/PinotSinkFunction.java

```diff
@@ -60,13 +63,19 @@ public class PinotSinkFunction<T> extends RichSinkFunction<T> implements Checkpo
   private TableConfig _tableConfig;
   private Schema _schema;
 
+  private String _segmentNamePrefix;
+
+  // Used to set creation time in segment name, if not provided, current time is used
+  @Nullable private Long _segmentUploadTimeMs;
+
   private transient SegmentWriter _segmentWriter;
   private transient SegmentUploader _segmentUploader;
   private transient ExecutorService _executor;
   private transient long _segmentNumRecord;
 
   public PinotSinkFunction(PinotGenericRowConverter<T> recordConverter, TableConfig tableConfig, Schema schema) {
     this(recordConverter, tableConfig, schema, DEFAULT_SEGMENT_FLUSH_MAX_NUM_RECORDS, DEFAULT_EXECUTOR_POOL_SIZE);
+    _segmentNamePrefix = DEFAULT_UPLOADED_REALTIME_SEGMENT_PREFIX;
```

Comment:
nit: this is redundant as the below ctor is already initialising  `_segmentNamePrefix`